benchmark-service/benchmark/benchmark.py

```diff
@@ -28,21 +73,51 @@ async def greet_user(request: web.Request) -> web.Response:
         'username': request.match_info.get('username', ''),
         'current_date': 'July 10, 2020'
     }
-    response = aiohttp_jinja2.render_template('user.html', request,
+    response = aiohttp_jinja2.render_template('name.html', request,
                                               context=context)
     return response
 
 
+@router.get('/name/{name}')
+@web_authenticated_developers_only(redirect=False)
+async def show_name(request: web.Request, userdata) -> web.Response:  # pylint: disable=unused-argument
+    benchmarks = get_benchmarks(default_filepath)
+    name_data = next((item for item in benchmarks['data'] if item['name'] == str(request.match_info['name'])),
+                     None)
+    try:
+        fig = px.scatter(x=list(range(0, len(name_data['times']))), y=name_data['times'])
```

Comment:
I think for the x axis you want a function that takes `name_data['times']` and converts it to something that looks like this:

`[0, 0, 0, 1, 1, 1, 2, 2, 2, 3, 3, 3, 4, 4, 4]`